### PR TITLE
Async/Await & misc.

### PR DESCRIPTION
**Pull request by vikRrrr**
- Switched from promises to ES7 async/await ~~u happy, Aether?~~ (I might have missed some error handling, looking forward to fix that)
- Change `rip` command to include mentions & raw args w/o duplicates
- Some other minor fix/changes

Note: if you're using a version of node lower than 7.6.0, you will have to pass the `--harmony` flag to node (and maybe even `--harmony_async_await`).

### CODE REVIEW

**Comment by vikRrrr** [comment]
Please wait for me to check if async/await error handling is complete before merging.

**Comment by vikRrrr** [comment]
Seems 👌

File: Markos.js [hunk 1]
```diff
@@ -1,55 +1,67 @@
...
     if (msg.author.id === config.owner) {
       try {
-        let dank = eval(args.join(' ')) // eslint-disable-line no-eval
+        const dank = eval(script) // eslint-disable-line no-eval
+
         msg.channel.sendMessage('**Result:**```js\n ' + dank + '\n```')
       } catch (e) {
         msg.channel.sendMessage(':warning: **ERROR** :warning: ```\n' + e + '\n```')
       }
-    } else if (msg.author.id === '116138050710536192') {
```
Comment (melmsie) [inline Markos.js]: I'd like to find a way to keep this. As this person kept trying to abuse the eval command.

Comment (vikRrrr) [inline Markos.js]: Alright. Maybe add a `blacklisted` object in the config?

i.e.
```json
...
  "blacklisted": {
    "eval": [ "116138050710536192" ]
  }
...
```

On a per-command basis, and maybe a global blacklist.

Comment (vikRrrr) [inline Markos.js]: Or do you really want to see when this specific user tried to use `eval`?

Comment (melmsie) [inline Markos.js]: I'll think it over. This'll be fine for now.

**Pull request merged**